skt/reporter.py

```diff
@@ -309,14 +309,15 @@ def getmergeinfo(self):
                       'repository at' % self.mergedata['base'][1][:12],
                       '  %s' % self.mergedata['base'][0]]
 
-        if not self.cfg.get("mergelog"):
-            cfgname = "config.gz"
-            result.append('\nThe kernel was built with the attached '
-                          'configuration (%s).' % cfgname)
-            self.attach.append((cfgname, gzipdata(self.mergedata["config"])))
-
         return result
 
+    def get_kernel_config(self, suffix=None):
+        cfgname = "config.gz" if not suffix else "config_{}.gz".format(suffix)
```

Comment:
I like that this syntax is compressed, but makes it a little difficult to read sometimes. I'm okay with it staying in, though.